### PR TITLE
[17.0][MIG] partner_bank_code : Migration to 17.0

### PR DESCRIPTION
**Pull request by peeranutjobgo**
I just chage ODOO version in __manifest__.py

### CODE REVIEW

**Comment by Saran440** [comment]
@peeranutjobgo Thank you for your contribution.

Please read how to migrated https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-17.0
for this pr. you forgot use command `git format-patch` for keep history.

**Comment by newtratip** [comment]
Pre-commit and test script still fail, could you check it ?

File: partner_bank_code/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Ecosoft Co., Ltd. (https://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Partner Bank Code",
+    "summary": "Add fields information in banks",
+    "version": "17.0.1.0.1",
```
Comment (newtratip) [inline partner_bank_code/__manifest__.py:7]:
```suggestion
    "version": "17.0.1.0.0",
```

**Comment by newtratip** [comment]
**Some comment**

- Bank code and bank branch code not display on name
Selection_2989
- Can not search bank by bank code or branch code (you can test by search bank via fill in bank code or branch code in menu Bank Account)
Selection_2990

In V.16, bank code and bank branch code display on name
Selection_2991

**Pull request closed without merging**